api/tests/devcomp/unit/infrastructure/datasources/learning-content/validation/element/video.js

```diff
@@ -6,7 +6,7 @@ const videoElementSchema = Joi.object({
   type: Joi.string().valid('video').required(),
   title: Joi.string().required(),
   url: Joi.string().uri().required(),
-  subtitles: Joi.string().required(),
+  subtitles: Joi.string().allow('').required(),
```

Comment:
On peut peut-être aussi rajouter `uri` ? 
```suggestion
  subtitles: Joi.string().uri().allow('').required(),
```